src/components/ReassignCenterModal.tsx

```diff
@@ -275,15 +308,46 @@ const {
         const selectedBlockCohortId = cohortDetails?.find(
           (item: any) => item?.type === "BLOCK"
         )?.cohortId;
-  
-        if (!selectedBlockCohortId) {
+        setSelectedBlockCohortIdForTL(selectedBlockCohortId)
+        // setFilters({
+        //   cohortId:selectedBlockCohortId,
+        //   role: Role.TEAM_LEADER,
+        //   status:[Status.ACTIVE]}
+        //   )
+const filters: FilterDetails=
+{
+  cohortId:selectedBlockCohortId,
+  role: Role.TEAM_LEADER,
+  status:[Status.ACTIVE]}
+console.log(filters)        
+      let limit=200;
```

Comment:
@AkshataKatwal16 why limit is 200?

Reply:
previously for cohortmember APi limit and offset is comulsory..but now it is not necessary